csrc/scheduler/matmul_heuristic.h

```diff
@@ -216,8 +241,7 @@ class MatmulParams : public HeuristicParams {
                                                            : "column-major")
        << "\n"
        << "Grid swizzle factor: " << grid_swizzle_factor << "\n"
-       << "Cluster dimensions: " << std::get<0>(cluster_dims) << " "
-       << std::get<1>(cluster_dims) << " " << std::get<2>(cluster_dims) << "\n"
+       << "Cluster dimensions: " << cluster_dims.toString() << "\n"
```

Comment:
Drop `Cluster dimensions` since it exists in `toString`
```suggestion
       <<  cluster_dims.toString() << "\n"
```